ui/components/multichain/network-list-item/network-list-item.tsx

```diff
@@ -163,13 +166,17 @@ export const NetworkListItem = ({
           backgroundColor={BackgroundColor.primaryDefault}
         />
       )}
-      <AvatarNetwork
-        borderColor={BorderColor.backgroundDefault}
-        backgroundColor={getAvatarNetworkColor(name)}
-        name={name}
-        src={iconSrc}
-        size={iconSize}
-      />
+      {isIconSrc(iconSrc) ? (
+        <Icon name={iconSrc} size={iconSize as IconSize} />
+      ) : (
+        <AvatarNetwork
+          borderColor={BorderColor.backgroundDefault}
+          backgroundColor={getAvatarNetworkColor(name)}
+          name={name}
+          src={iconSrc}
+          size={iconSize as AvatarNetworkSize}
+        />
+      )}
```

Comment:
Allows this component to render either a network image, or an icon (required for the new design in figma)